themes/backdrop/static/css/backdrop.css

```diff
@@ -7559,6 +7559,7 @@ footer {
         width: 33.3333333333%;
         float: left;
         min-height: auto;
+        overflow: auto;
```

Comment:
Great, thanks, this is important issue to be solved asap, but it seems this solution produces a double scroll bar. Also, logos are still not visible when scrolling only through the main body area, as you need to specifically scroll in the right pane area to see them.

I think I will add a few additional lines to solve this, and we can merge all together.